src/geanyobject.c

```diff
@@ -52,6 +56,11 @@ struct _GeanyObjectPrivate
 	gchar dummy;
 };
 
+/** Get the GObject-derived GType for GeanyObject
+ *
+ * @return GeanyObject type */
+GEANY_API_SYMBOL
+GType geany_object_get_type(void);
```

Comment:
that too, it's totally useless but for girscanner, so make it @gironly at least (and better, it should be private)